extensions/lifecycle/tasks/LifecycleTask.js

```diff
@@ -575,18 +573,18 @@ class LifecycleTask extends BackbeatTask {
      * date
      * @return {object} The most applicable transition rule
      */
-    _getStoreTransition(params) {
+    _compareTransitions(params) {
         const { transition1, transition2, lastModified } = params;
         if (transition1 === undefined) {
             return transition2;
         }
         if (transition2 === undefined) {
             return transition1;
         }
-        const delta =
-            this._getTransitionTimestamp(transition1, lastModified) -
-            this._getTransitionTimestamp(transition2, lastModified);
-        return delta > 0 ? transition1 : transition2;
+        // Which transition is the most recent?
+        return this._getTransitionTimestamp(transition1, lastModified) -
```

Comment:
it could be simply `this._getTransitionTimestamp(...) > this._getTransitionTimestamp(...) ? ...`